specification/specs/x_pack/security/privileges/get_privileges/GetPrivilegesRequest.ts

```diff
@@ -24,8 +24,8 @@
  */
 interface GetPrivilegesRequest extends RequestBase {
   path_parts?: {
-    application?: Name
-    name?: Name
+    application?: Name | boolean
+    name?: Name | boolean
```

Comment:
This looks very weird, the [resta-api-spec](https://github.com/elastic/elasticsearch/blob/7.x/x-pack/plugin/src/test/resources/rest-api-spec/api/security.get_privileges.json) never mentions the boolean type, nor the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-privileges.html).

I tried to run this test on my machine, but it passed. Can you paste here the filename of the failing test?
I guess that the yaml tests are doing something weird.